pkgs/applications/kde/kirigami-gallery.nix

```
@@ -0,0 +1,31 @@
+{ mkDerivation
+, lib
+, kirigami2
+, extra-cmake-modules
+, kitemmodels
+, qtgraphicaleffects
+, qtquickcontrols2
+, qttools
+, qtbase
+}:
+
+mkDerivation rec {
+  pname = "kirigami-gallery";
+
+  nativeBuildInputs = [ extra-cmake-modules qttools ];
+
+  buildInputs = [
+    qtgraphicaleffects
+    qtquickcontrols2
+    kirigami2
+    kitemmodels
+  ];
+
+  meta = with lib; {
+    homepage = "https://apps.kde.org/kirigami2.gallery/";
+    description = "View examples of Kirigami components";
+    license = licenses.lgpl2;
+    maintainers = with maintainers; [ shadowrz ];
+    broken = lib.versionOlder qtbase.version "5.15";
```

`lib` is already in scope:
```suggestion
    broken = versionOlder qtbase.version "5.15";
```